packages/components/src/components/buttons/buttonStyleUtils.ts

```diff
@@ -1,10 +1,25 @@
 import { css, DefaultTheme } from 'styled-components';
-import { spacings, spacingsPx } from '@trezor/theme';
+import {
+    BackgroundTertiaryElevationColor,
+    Elevation,
+    spacings,
+    spacingsPx,
+    nextElevation,
+} from '@trezor/theme';
 
 export type ButtonVariant = 'primary' | 'secondary' | 'tertiary' | 'destructive';
 export type ButtonSize = 'large' | 'medium' | 'small' | 'tiny';
 export type IconAlignment = 'left' | 'right';
 
+export const mapElevationToBackgroundTertiary: Record<Elevation, BackgroundTertiaryElevationColor> =
+    {
+        '-1': 'backgroundTertiaryElevationNegative', // For example left menu is negative elevation
+        0: 'backgroundTertiaryDefaultOnElevation0',
+        1: 'backgroundTertiaryDefaultOnElevation1',
+        2: 'backgroundTertiaryDefaultOnElevation2',
+        3: 'backgroundTertiaryDefaultOnElevation3',
+    };
```

Comment:
nit: Since that object _(and some others)_ exists solely for mapping `-1` to `Negative`, how about just having 

```
const mapElevation = {
    '-1': 'Negative',
    0: '0',
    1: '1',
    2: '2',
    3: '3',
};
```

And then 

```
background: ${({ theme }) => theme[`backgroundTertiaryDefaultOnElevation${mapElevation[elevation]}`]};
```

It's not shorter but allows to see in place what color token is actually used and we won't have to create the elevation map for every property 🫨

_cc: @peter-sanderson_

Reply:
I am not sure if typescript would work the same way. There is some way in TS how to concatenate strings in type-safe way, but not sure it it wold add to readability even if it works